src/utils/fhir.js

```diff
@@ -324,7 +324,10 @@ function postSubscriptionsToEHR(subscriptions) {
 
       // Create/Update Subscriptions on EHR server
       const ehrToken = await getAccessToken(ehrServer.endpoint);
-      const headers = { Authorization: `Bearer ${ehrToken}` };
+      const headers = {
+        Authorization: `Bearer ${ehrToken}`,
+        'Content-Type': 'application/fhir+json'
```

Comment:
Is it worth setting this conditionally based on the metadata? I haven't actually tested with the medmorph-fhir-server yet but I think it should work. I could imagine other servers being as picky as the subscription-proxy and requiring `application/json` instead of `application/fhir+json`.

Reply:
Can we reliably pick it from metadata? The Epic open sandbox metadata (https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/R4/metadata) includes `json` (alongside `xml`) as a supported format, not `application/...`. I do see that HAPI lists the full mime type there.

Reply:
So from the CapabilityStatement resource it looks like the correct syntax is just `json` or `xml`, which doesn't help us determine whether or not to use `fhir+`